pkg/exec/builder/execute_test.go

```diff
@@ -145,6 +145,10 @@ func TestExecuteStep_HasOrderedArguments(t *testing.T) {
 }
 
 func TestExecuteStep_SpecifiesCustomWorkingDirectory(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip() // pwd not available on windows (for some reason the only command that works is "help" ?!?)
```

Comment:
would love any help on this, all commands I tried except `help` resulted in error stating "no such command on %path"

Reply:
Is this running in context of powershell or cmd?  `cd` or `chdir` would work on cmd and I think `pwd` is an alias in powershell that will list the current directory

Reply:
Ah I found the problem, they are not executables, they are internal commands. Need to execute them using `cmd.exe /c dir` for example. https://stackoverflow.com/a/71639965